src/utils/constants.js

```diff
@@ -1,53 +1,40 @@
-export const initialCards = [
-  {
-    name: "Yosemite Valley",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/yosemite.jpg",
-  },
-  {
-    name: "Lake Louise",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/lake-louise.jpg",
-  },
-  {
-    name: "Bald Mountains",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/bald-mountains.jpg",
-  },
-  {
-    name: "Latemar",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/latemar.jpg",
-  },
-  {
-    name: "Vanoise National Park",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/vanoise.jpg",
-  },
-  {
-    name: "Lago di Braies",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/lago.jpg",
-  },
-];
-//----------------------------------------------------------------------------------------
-//                                      Elements
-//----------------------------------------------------------------------------------------
-
 const profileEditButton = document.querySelector(".profile__edit-button");
 const profileEditModal = document.querySelector("#profile-edit-modal");
+const avatarEditModal = document.querySelector("#avatar-edit-modal");
 const addCardModal = document.querySelector("#add-card-modal");
+
 const profileModalCloseButton = profileEditModal.querySelector("#modal-close");
 const addCardModalCloseButton = addCardModal.querySelector("#modal-close");
+
 const profileTitle = document.querySelector(".profile__title");
 const profileDescription = document.querySelector(".profile__description");
+
 const profileTitleInput = document.querySelector("#profile-title-name");
 const profileDescriptionInput = document.querySelector(
   "#profile-description-input"
 );
+
 const addNewCardButton = document.querySelector(".profile__add-button");
+
+//forms
 const profileEditForm = profileEditModal.querySelector("#modal-form");
```

Comment:
**COULD BE IMPROVED**

We can get any form using their **unique** attribute `name` or `id` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

You need to add `name` or `id` first in `index.html` to achieve that.

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms